src/command/dev/do_dev.rs

```
@@ -42,6 +43,7 @@ impl Dev {
         client_config: StudioClientConfig,
         log_level: Option<Level>,
     ) -> RoverResult<RoverOutput> {
+        dotenv().ok();
```

Just want to confirm this won't throw an error if the `.env` file does not exist? I'm assuming no because we don't have the `?` character like in the documentation

> `// Load environment variables from .env file.`
> `// Fails if .env file not found, not readable or invalid.`
> `dotenvy::dotenv()?;`

---

@swcollard You're right! `ok()` discards the error if any.

---

You got me worried. So, I just tested it manually. All look good! 😄